doc/lua_api.txt

```diff
@@ -5323,8 +5321,9 @@ This is basically a reference to a C++ `ServerActiveObject`
         text = "My Nametag",
       }
 
-#### LuaEntitySAO-only (no-op for other objects)
+#### Lua entity only (no-op for other objects)
```

Comment:
`lua entity` in Lua has the `object` property to reference this "class". The correct term here would be `lua object`, although this term is nowhere else used in this file.

Reply:
How about
`Lua entity object only`
?

Reply:
I think the PR is ok as it is, `(no-op for other objects)` implies it is an object. `entity` is clearer to understand for modders and keeps the section title similar.

Reply:
I agree with paramat. "Lua object" sounds quite confusing, as we've referred to these as Lua entities everywhere else.